### PR TITLE
Add resources tab

### PR DESCRIPTION
**Pull request by kiiadi**
Screen Shot 2020-10-16 at 5 11 59 PM

### CODE REVIEW

**Review by hunterwerlla — COMMENTED** [review]
Good add, why didn't we have this? Can you also edit the UI test? sorry and thanks

**Comment by kiiadi** [comment]
Added UI tests

File: ui-tests/tst/software/aws/toolkits/jetbrains/uitests/tests/CloudFormationBrowserTest.kt [hunk 1]
```diff
@@ -116,6 +123,9 @@ class CloudFormationBrowserTest {
         waitForStackDeletion()
     }
 
+    private fun IdeaFrame.clickOn(tab: String) {
+        findAndClick("//div[@accessiblename='$tab' and @class='JLabel' and @text='$tab']")
+    }
     private fun IdeaFrame.clickOnOutputs() {
```
Comment (hunterwerlla) [inline ui-tests/tst/software/aws/toolkits/jetbrains/uitests/tests/CloudFormationBrowserTest.kt:129]: nit clickOnOutputs/Events is now dead code

**Review by hunterwerlla — APPROVED** [review]
Thanks for adding a UI test!

File: ui-tests/tst/software/aws/toolkits/jetbrains/uitests/tests/CloudFormationBrowserTest.kt (hunk 1 above)
Comment (kiiadi) [reply ui-tests/tst/software/aws/toolkits/jetbrains/uitests/tests/CloudFormationBrowserTest.kt:129]: yeah its actually removed in #2165

Comment (hunterwerlla) [reply ui-tests/tst/software/aws/toolkits/jetbrains/uitests/tests/CloudFormationBrowserTest.kt:129]: Yeah I saw that after making this comment, looks good then

**Pull request merged**